### PR TITLE
Add a clear warning to the top of the old language tour

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

File: src/_guides/language/language-tour.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ canonical: https://dart.dev/language
 noindex: true
 ---
 
+{{site.alert.warning}}
+  The Dart language tour's content has been split across
+  several different pages in the sidenav under **Language**.
```
Comment (atsansone) [inline src/_guides/language/language-tour.md:12]: **issue:** `sidenav` is jargon and doesn't translate. Consider:
```suggestion
  several different pages in the left side navigation under **Language**.
```

**Review by atsansone — CHANGES_REQUESTED** [review]
Requesting one change for readability and accessibility.

**Review by MaryaBelanger — APPROVED** [review]
LGTM w/ Tony's change

File: src/_guides/language/language-tour.md (hunk 1 above)
Comment (parlough) [reply src/_guides/language/language-tour.md]: Thanks, I've made the change!

**Pull request merged**